src/vignette/media_types.clj

```diff
@@ -15,6 +15,10 @@
   [file]
   (contains? webp-mime-types (mime-type-of (or file ""))))
```

Comment:
As you added the `webp-supported-content-type?`, we should either call it here like `(webp-supported-content-type? (mime-type-of (or file "")))` or make change other places to use `webp-supported-content-type?` instead of `webp-supported?`